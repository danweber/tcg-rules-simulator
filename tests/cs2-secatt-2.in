VERSION:2
#   "evolveEffect": "\uff1cSecurity Attack +1\uff1e",
FIELD2:Attacker,DW1-Van,CS2-09,CS2-09
SECURITY1:CS1-02 CS1-02 CS2-Excalibur CS1-02
GAME:P2 T1 MAIN 7
DUMP:SECURITY,TRASH

ECHO: -- attack into sec, after second check the attacker dies
CMD:ATTACK Attacker Player
STEP:AUTOANSWER:CAPTURE
DUMP:SECURITY,TRASH

EXIT
