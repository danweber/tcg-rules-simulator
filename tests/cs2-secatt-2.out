FIELD1:
FIELD2:Attacker,DW1-Vanillamon,CS2-Masamune,CS2-Masamune
SECURITY1:CS1-02 CS1-02 CS2-Excalibur CS1-02
SECURITY2:
TRASH1:
TRASH2:
GAME:P2 T1 MAIN 7
ECHO: -- attack into sec, after second check the attacker dies
FIELD1:
FIELD2:
SECURITY1:CS1-02 CS1-02
SECURITY2:
TRASH1:CS1-02 CS2-Excalibur
TRASH2:CS2-Masamune CS2-Masamune DW1-Vanillamon
GAME:P2 T1 MAIN 7
